_includes/02-image.md

```diff
@@ -0,0 +1 @@
+Replace this with an image, like your photo. Ensure you include some alt-text.
```

Comment:
! [ pic of Bob] ( https://upload.wikimedia.org/wikipedia/commons/4/48/Rapper_B.o.B_2013.jpg)

Reply:
yep, lets do it